webplugin/js/app/km-richtext-markup-1.0.js

```diff
@@ -143,12 +143,12 @@ getButtonTemplate:function(options,requestType, buttonClass){
 getQuickRepliesTemplate:function(){
     return`<div class="km-cta-multi-button-container">
             {{#payload}}
-                 <button aria-label="{{title}}" title='{{message}}' class="km-quick-replies km-custom-widget-text-color {{buttonClass}} " data-metadata = "{{replyMetadata}}">{{title}}</button>
+                 <button aria-label="{{title}}" title='{{message}}' class="km-quick-replies km-custom-widget-text-color {{buttonClass}} " data-metadata = "{{replyMetadata}}" data-languageCode = "{{updateLanguage}}">{{title}}</button>
```

Comment:
@Suraj-Kumar below it is {{action.updateLanguage}} here it is {{updateLanguage}}, both are supposed to be different or same?

Reply:
both are different depends on JSON being passed.

Reply:
is it json format specified by us somewhere or is it through dialogflow, trying to understand what makes this different than getGenericSuggestedReplyButton.
If it is enforced by us then why can't keep it consistent?

Reply:
getGenericSuggestedReplyButton  was added to support different type of buttons in single message. So the JSON format is slightly different there. once this template is added into mobile SDKs will remove old JSON from docs and promote this one.